esp/logging/logginglib/logthread.cpp

```diff
@@ -181,9 +182,14 @@ bool CLogThread::queueLog(IEspUpdateLogRequest* logRequest)
 
 bool CLogThread::queueLog(IEspUpdateLogRequestWrap* logRequest)
 {
-    unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
+// ESP Log Update
+// Would be good to find a way to delay calculation of delta time in case log message is filtered
+// Look for msTick() - startTime elsewhere
+//    unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
+    unsigned startTime = msTick();
```

Comment:
Likely ok, but is the overhead of the msTick call warranted? @wangkx -thoughts?

Reply:
The msTick call does add the overhead. I wish we can query the log level here.

Reply:
We can. It's just call a threshold now.